src/cmd/run.rs

```diff
@@ -121,38 +115,26 @@ struct CmdContext<D> {
     /// The database connection.
     db: D,
 
-    /// The semaphore for global concurrency of integrations.
-    concurrency: Semaphore,
-
     /// The location of FOSSA CLI
     cli: Location,
 }
 
-impl<D> CmdContext<D> {
-    /// Acquire a concurrency permit from the global limiter.
-    pub async fn acquire_permit(&self) -> Result<SemaphorePermit<'_>, Error> {
-        self.concurrency
-            .acquire()
-            .await
-            .context(Error::AcquireConcurrencyPermit)
-            .describe("Broker limits concurrent integration operations to the value specified in the config file.")
-            .help("this is almost definitely a temporary condition or a bug, restarting Broker may resolve the issue")
-    }
-}
-
 /// The primary entrypoint.
 #[tracing::instrument(skip_all, fields(subcommand = "run"))]
 pub async fn main<D: Database>(ctx: &AppContext, config: Config, db: D) -> Result<(), Error> {
+    warn!(
+        concurrency = config.concurrency(),
+        "the concurrency setting is temporarily ignored, but we plan to support it again in a future release."
```

Comment:
[nit] Other warns in this file have the first word capitalized and don't end in `.` unless there are multiple sentences

Reply:
oops, thanks